packages/twenty-server/src/modules/messaging/message-import-manager/services/messaging-send-message.service.ts

```diff
@@ -93,6 +93,9 @@ export class MessagingSendMessageService {
         await microsoftClient.api(`/me/messages/${response.id}/send`).post({});
         break;
       }
+      case ConnectedAccountProvider.IMAP: {
+        throw new Error('IMAP provider does not support sending messages');
```

Comment:
<img width="698" alt="Screenshot 2025-06-13 at 09 32 49" src="https://github.com/user-attachments/assets/53a20566-6876-4287-b43b-771287d04a94" />

It's a bit deceptive that I'm able to pick it in workflows. If we implement SMTP as a fast follow before releasing betta I think it's fine but we could not ship to prod without it